index.js

```diff
@@ -13,5 +13,20 @@ module.exports = (robot, _, Settings = require('./lib/settings')) => {
     if (defaultBranch && settingsModified) {
       return Settings.sync(context.github, context.repo())
     }
+    else if (!defaultBranch && settingsModified) {
```

Comment:
Remove settingsModified as this condition will not work if it is there

Reply:
@knrt10 Why do you want to remove `settingsModified` ?
We only want to create a status if the settings are modified.
There is no point if we create a status without the settings being changed. :)

Reply:
Here settings modified will always return undefined if you create changes in any other branch other than master or default branch. You can try and see in console.log. Hope this helps

Reply:
I am sorry I was working on a different repo so my output was not coming. You are right. My bad.